backend/src/controllers/words/words.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response } from "express";
+import { WordsService } from "../../services/words/words.service";
+
+export class WordsController {
+  static async getWords(req: Request, res: Response) {
+    try {
+      const words = await WordsService.getWords(
+        req.body.owner_id,
```

Comment:
it would be nice to add validation layer for the parameters received, for example you could throw a 400 Bad Request for owner_id that is a text over 100 characters etc. that'll save you some unnecessary queries to database as well as help the client know what's expected

status 500 is usually used only for extreme cases, you as a developer handle all what you can think of (wrong data, bad format, not found, entity already existing etc - it's all some [HTTP code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status))

Reply:
I see an option here for i.e 400 and 404 errors (if you include logging service and privatize data, then 403 error is also possible to happen)